Sources/AsyncHTTPClient/ConnectionPool.swift

```diff
@@ -179,6 +179,19 @@ final class ConnectionPool {
             }
             return config
         }
+
+        var description: String {
+            var hasher = Hasher()
```

Comment:
Are you sure this is what you want? This hash is not stable across executions.

Reply:
I think I am... It is for aggregation within one run. I don't think I want to look at connection pool state changes between runs.

Reply:
Great, then this all looks good.